### ISSUE TITLE
BC break (regression?): injectDOMPropertyConfig

### ISSUE THREAD

**Issue by indeyets** [FormidableLabs/rapscallion#102]
`cat test.js`

```
import rapscallion from 'rapscallion';
import dom from 'react-dom';
```

`./node_modules/.bin/babel-node test.js`

```
…/node_modules/fbjs/lib/invariant.js:51
    throw error;
    ^

Invariant Violation: injectDOMPropertyConfig(...): You're trying to inject DOM property 'aria-current' which has already been injected. You may be accidentally injecting the same DOM property config twice, or you may be injecting two configs that have conflicting property names.
    at invariant (…/node_modules/fbjs/lib/invariant.js:44:15)
    at Object.injectDOMPropertyConfig (…/node_modules/react-dom/lib/DOMProperty.js:73:99)
    at Object.inject (…/node_modules/react-dom/lib/ReactDefaultInjection.js:69:30)
    at Object.<anonymous> (…/node_modules/react-dom/lib/ReactDOM.js:27:23)
    at Module._compile (module.js:569:30)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .js] (…/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (…/node_modules/react-dom/index.js:3:18)
    at Module._compile (module.js:569:30)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .js] (…/node_modules/babel-register/lib/node.js:152:7)
```

**Comment by indeyets** [comment]
@fleg ping. this seems to be related to recent changes

**Comment by fleg** [comment]
ok, I look into it as soon as possible

**Comment by fleg** [comment]
meanwhile you can simply change order of imports
```
import dom from 'react-dom';
import rapscallion from 'rapscallion';
```
I guess it will help

**Comment by indeyets** [comment]
@fleg sure, but example is artificial. in the real world this means I have to explicitly import react-dom in my render module where I never had it. Currently it is present strictly in components which need it

that's why it is a BC break

**Comment by divmain** [comment]
Resolved in `v2.1.10`.
